audb/core/load.py

```diff
@@ -1225,32 +1289,37 @@ def load_table(
     with FolderLock(db_root):
 
         # Start with database header without tables
-        db, backend = load_header(
+        db, backend = load_header_to(
             db_root,
             name,
             version,
         )
 
         # Load table
-        table_file = os.path.join(db_root, f'db.{table}')
-        if not (
-                os.path.exists(f'{table_file}.csv')
-                or os.path.exists(f'{table_file}.pkl')
-        ):
-            _load_tables(
-                [table],
-                backend,
-                db_root,
-                db,
-                version,
-                None,
-                deps,
-                Flavor(),
-                cache_root,
-                num_workers,
-                verbose,
-            )
-        table = audformat.Table()
-        table.load(table_file)
+        tables = (
```

Comment:
```suggestion
        tables = _misc_tables_used_in_scheme(db) + [table]
```

Reply:
Sorry, wanted to create a create multi-line comment, but failed again...

Reply:
Changed it manually.